hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/client/ReplicatedMapPortableHook.java

```diff
@@ -41,7 +41,7 @@
     public static final int VALUES_COLLECTION = 14;
     public static final int MAP_ENTRY_EVENT = 18;
 
-    private static final int LENGTH = 21;
+    private static final int LENGTH = MAP_ENTRY_EVENT + 1;
```

Comment:
I don't think this will change anything, just make the `constructors` array a bit smaller, since it anyway returns `null` for IDs which are not found (and there is a proper length check in the create method).